### ISSUE TITLE
Video and Audio Lag on videos longer than 5 minutes

### ISSUE THREAD

**Issue by greenman86** [yuliskov/SmartYouTubeTV#74]
Using a firestick which runs over video content perfectly fine also

**Comment by andrewtjb** [comment]
Just thought id say ive been playing about trying to fix this problem with my 1st and 2nd gen firestick and ive seemed to have fixed the lag. Just go into the laucher setting and limit the codec to avc1 1080p then select the 4k version and playback should be perfect.

**Comment by greenman86** [comment]
Hi Andrew,

Yeh I'm using the first 4k option and it's working flawlessly.

Thanks for the follow up

**Comment by zakir1988** [comment]
Using 1080 main launcher for youtube I'm using firestick 2nd generation where the video lags or stutters after a while then the video stops playing says loading can't exit or do anything have to restart the device. 

That's the only problem I have with the app I don't know if I selected the right launcher for right device.

I think you should tell us which launcher would be right one to select for the devices for example select 1080 main for android boxes and 1st, 2nd gen firestick, select 4k main if your devices supports 4K etc.... that will emulate any problems or laging etc...

**Comment by yuliskov** [comment]
> I think you should tell us which launcher would be right one to select for the devices

that is a case. I don't know exactly. Someone says that 4K is the best option for the firestick, while others prefer to use simple 1080 launcher.

**Comment by zakir1988** [comment]
UK, USA and International have diffrent Firestick firmware it should not effect the way apps play as the specs are same for each device for example USA version Firestick has same specs as the UK one only diffrence is firmware with added extra features thats all. 

Here is the specs for amazon fire tv devices in the  link given https://www.anandtech.com/show/13423/amazon-launches-fire-tv-stick-4k

You'll get a better understadding rather then firestick users complaining.

**Comment by yuliskov** [comment]
@zakir1988 seems you're right. But I still need a user's feedback since I don't have such devices on hands.

**Comment by zakir1988** [comment]
Yes i did some testing problems i found is that 1080 main and 1080 alt not as stable.

1080 alt  video stutters sometimes and i have told you about the 1080 main.

4k main works very well on the firestick and 4k alt.

Forgot to say 1080 alt when it first launches on the firestick when we do a search the firestick keyboard popsup then we have to press back then can use the YouTube search, when firestick keyboard popsup we can still use it to search it inputs it into the app but only comes up once we launch the app.

Hope you find this information good.

**Comment by greenman86** [comment]
Hi there,

I noticed this on my first gen firestick also. 4K options are the best ones
to use. The 1080p options eventually freezes after a few minutes of video
play

**Comment by zakir1988** [comment]
4K launcher will be good for The firestick 4K always select the right luncher dpending on TV resolution but people are using 4K luncher on firestick 2nd generation its stable work with no problem.

4K resolution is not supported by amazon on the older firestick but works a charm.

**Comment by greenman86** [comment]
The update you released today isnt installing "there's a problem parsing
this package" is what i get on my Firestick

**Comment by crcerror** [comment]
@yuliskov Is it posssible to reimplent the old 720p option? This was possible to slelect in v6.12.58 and worked fine. If I select the 1080 option in latest release (v6.15.98_latest) I receive stutterings after 30min play on 1.Gen ATV-Stick

**Comment by yuliskov** [comment]
@crcerror old 720p option has lowered resolution by default. I think, you should try to run the latest release with lower video resolution.

**Comment by greenman86** [comment]
the auto update yuriy

**Comment by yuliskov** [comment]
@greenman86 autoupdate will be fixed soon. If you don't want to wait, here the direct link: https://github.com/yuliskov/SmartYouTubeTV/releases/download/stable/smartyoutubetv_latest.apk

#194

**Comment by yuliskov** [comment]
@crcerror already fixed in the release above. It's totally my fault.

**Comment by crcerror** [comment]
@yuliskov No the failure with the exit button has something to do with firmware changes. I used v6.12.58 all the time (with disabled autoupdate) and it worked fine till the last days.

The problems accoured as the exit dialog (YES/NO-box) changed with the graphical slide menu....

Just saw your edit .... so it's the web-api okay :+1: 

Thank you for clearance

**Comment by pannal** [comment]
With the 1080p profiles I have a slight audio/video delay on my 2nd gen firetv stick. 4K profiles work well once I limit the codes to avc. 

Thank you for this great piece of software!
